docs/source/model_import.rst

```diff
@@ -0,0 +1,40 @@
++==================================================+===================+===========================================+
+| :class:`xgboost.XGBClassifier`                   | PMML, ONNX, JSON  | :class:`snapml.BoostingMachineClassifier` |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`xgboost.XGBRegressor`                    | PMML, ONNX, JSON  | :class:`snapml.BoostingMachineRegressor`  |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`lightgbm.LGBMClassifier`                 | PMML, ONNX, Text  | :class:`snapml.BoostingMachineClassifier` |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`lightgbm.LGBMRegressor`                  | PMML, ONNX, Text  | :class:`snapml.BoostingMachineRegressor`  |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`snapml.BoostingMachineClassifier`        | PMML              | :class:`snapml.BoostingMachineClassifier` |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`snapml.BoostingMachineRegressor`         | PMML              | :class:`snapml.BoostingMachineRegressor`  |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`sklearn.ensemble.RandomForestClassifier` | PMML, ONNX        | :class:`snapml.RandomForestClassifier`    |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`sklearn.ensemble.RandomForestRegressor`  | PMML, ONNX        | :class:`snapml.RandomForestRegressor`     |
++--------------------------------------------------+-------------------+--------+----------------------------------+
+| :class:`sklearn.ensemble.ExtraTreesClassifier`   | PMML, ONNX        | :class:`snapml.RandomForestClassifier`    |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`sklearn.ensemble.ExtraTreesRegressor`    | PMML, ONNX        | :class:`snapml.RandomForestRegressor`     |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`snapml.RandomForestClassifier`           | PMML              | :class:`snapml.RandomForestClassifier`    |
++--------------------------------------------------+-------------------+-------------------------------------------+
+| :class:`snapml.RandomForestRegressor`            | PMML              | :class:`snapml.RandomForestRegressor`     |
++--------------------------------------------------+-------------------+-------------------------------------------+
+
+.. autofunction:: snapml.import_model
```

Comment:
would it make sense to mention that for the SnapML models, they could be saved as pkl files as well for when the file model size is required to be small? in which case pickle should be used for saving/loading the model instead of the import_model function?

Reply:
It might make sense to add a paragraph explaining that pkl is the normal way but that PMML can be used to train on x86 and deploy on Z (i.e., to overcome endianness issues).

Reply:
right, maybe we could add such a note, as you suggested above

Reply:
I added a paragraph as discussed.